src/OpenLoco/Windows/TileInspector.cpp

```diff
@@ -77,7 +75,7 @@ namespace OpenLoco::Ui::Windows::TileInspector
 
     static void activateMapSelectionTool(Window* const self)
     {
-        Input::toolSet(self, widx::panel, 42);
+        Input::toolSet(self, widx::panel, CursorId::crosshair);
```

Comment:
The tile inspector was using the placeSignal cursor before. I hadn't noticed before, as cursors weren't working on Linux until recently :sweat_smile: